docs/Developer Reference/getting-help.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Getting Support"
+permalink: "docs/getting-help/"
+---
+### Where do I go to get help and support? 
+
+If you run into issues and the available documentation, videos, and code repositories are not able to assist you, the best way to get help is to follow this support escalation process. Start with step 1, and continue to the next till you reach a solution. Note, sometimes, you might be asking a question that is too theoretical, or hasn't been solved, so you might not always get what you're looking for!
+
+## 1. Double check the documentation
+
+Check to see if you missed any code, documentation, blog, or video on the topic or issue you're looking for. There are typically a few different resources on a topic if one doesn't answer exactly what you're looking for. You can also use the documentation search bar to look up things as well. 
+
+## 2. Do a web search for the specific error or situation you're in. 
+
+Often times, someone else has asked the question that you're asking. If you copy paste the error into the google or web search bar, there is a good chance that some helpful material will come up. 
+
+## 3. Make an issue on the github or code repository
+
+This is only applicable if you're working with a certain set of code. For example, if you're having an issue working with the [Chainlink Hardhat Starter kit](https://github.com/smartcontractkit/hardhat-starter-kit), maybe you can make an issue on the repo explaining exactly what's going on. 
+
+When writing issues, remember to:
+- Keep titles short
+- Remain clear and concise
+- Format your message ([using 3 backticks](https://discuss.codecademy.com/t/using-backticks-to-format-your-code/3697) will help format your code)
```

Comment:
```suggestion
- Format your issue description. Use [three backticks (```)](https://discuss.codecademy.com/t/using-backticks-to-format-your-code/3697) to format format your code or log output.
```